types/jest/index.d.ts

```diff
@@ -61,6 +61,14 @@ declare namespace jest {
      */
     function resetAllMocks(): typeof jest;
     /**
+     * available in Jest 20.1.0+
```

Comment:
Actually, this function was added in 20.1.0 but exposed in 21.1.0, that said, you'll get "jest.restoreAllMocks is not a function" error before 21.1.0, should bump version to 21.1 as well.

Ref: [Jest v21.1.0](https://github.com/facebook/jest/blob/master/CHANGELOG.md#jest-211)
- Expose restoreAllMocks to object (facebook/jest#4463)

Reply:
Opened https://github.com/facebook/jest/pull/4522 to fix docs on the Jest website.

Reply:
hey ikatyang, thx for your review and feedback.